requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Framework must-haves
 numpy>=1.16.4
-torch>=1.1.0
+torch<=1.5
```

Comment:
could you please return `torch>=1.1.0` we are already working on torch version issue